apps/emqx_conf/src/emqx_conf_app.erl

```diff
@@ -175,7 +175,7 @@ copy_override_conf_from_core_node() ->
                 _ ->
                     [{ok, Info} | _] = lists:sort(fun conf_sort/2, Ready),
                     #{node := Node, conf := RawOverrideConf, tnx_id := TnxId} = Info,
-                    HasDeprecatedFile = maps:get(has_deprecated_file, Info, false),
+                    HasDeprecatedFile = has_deprecated_file(Info),
```

Comment:
There is no need to introduce a new version bpapi.
All we need to do is to check `maps:is_key(has_deprecated_file, Info)`

Reply:
We will always get false, regardless of whether the old **version(v5.0.22) has this file. We can't change old code.

Reply:
Mix versions in the same cluster for a long time is not normal.

1. v5.0.22 has cluster-override.conf, v5.0.24 join in, v5.0.24 will copy from v5.0.22 without has_deprecated_file.
So the v5.0.24 will write to cluster.hocon. result: after all node upgrade to v5.0.24,  the config priority is changed.
So we need rpc to old node to check the file exist.

2. v5.0.22 don't have cluster-override.conf, v5.0.24 join in, v5.0.24 will copy from v5.0.22 without has_deprecated_file. So the v5.0.24 will write to cluster.hocon. after all node upgrade to v5.0.24, the config priority is changed, This level of change is normal because it had no priority on the old version node.
It is important to note that when rolling upgrades (when there are multiple versions) we should not change the configuration via the old version without cluster-override.conf.